packages/flutter_tools/test/commands/create_test.dart

```diff
@@ -278,7 +279,7 @@ void main() {
         'ios/Classes/FlutterProjectPlugin.h',
         'ios/Classes/FlutterProjectPlugin.m',
         'ios/Classes/SwiftFlutterProjectPlugin.swift',
-        'lib/flutter_project.dart',
+        'lib/flutter_project.dart'
```

Comment:
nit: we likely still want the trailing comma

Reply:
Yes, that was a typo. I've restored it.